src/Option.js

```diff
@@ -46,11 +53,13 @@ class Option extends Component {
   }
 
   handleSelectOption = (e) => {
-    this.props.onSelect(this.props.value);
+    this.props.onSelect(getOptionValue(this.props.option));
   }
 
   render() {
-    const { label, isFocus } = this.props;
+    const { option, isFocus } = this.props;
+
+    const optionRenderer = this.props.customRenderOption || defaultRenderOption;
```

Comment:
ตอนนี้ defaultRenderOption เป็น static ที่ประกาศอยู่ด้านบนลองเพิ่ม
function optionRender(){
   const { label, value, customRenderOption  } = this.props;
   return customRenderOption(label, value) || (<span>{label}</span>);
}